src/porepy/fracs/non_conforming.py

```diff
@@ -473,12 +473,25 @@ def update_cell_faces(g, delete_faces, new_faces, in_combined, fn_orig,
     # relations
     if in_combined[0] < in_combined[-1]:
         for i in range(deleted_2_new_faces.size):
-            deleted_2_new_faces[i] = new_faces[np.arange(in_combined[i],
-                                                         in_combined[i+1])]
+            if in_combined[i] == in_combined[i+1]:
```

Comment:
Please add comment on what happens here. It is difficult enough, without having to guess what we were thinking months before.

Reply:
I already forgot :(